script/reproducible/verify.bat

```diff
@@ -0,0 +1,129 @@
+@ECHO OFF
+set CHECK_VERSION=%1
+SHIFT
+
+set buildir=C:\projects\
+
+REM Interpret arguments
+:loop
+IF NOT "%1"=="" (
+    IF "%1"=="-buildir" (
+        SET buildir=%2
+        SHIFT
+    )
+    SHIFT
+    GOTO :loop
+)
+
+@ECHO ON
+
+set ELECTRON_OUT_DIR=Default
+set ELECTRON_ENABLE_STACK_DUMPING=1
+
+REM Check for disk space
+Rem        543210987654321
+Set "Blank=               "
+Set "GB100=   107374182400"
+
+for /f "tokens=2" %%A in (
+  'wmic LogicalDisk Get DeviceID^,FreeSpace ^| find /i "C:"'
+) Do Set "FreeSpace=%Blank%%%A"
+Set "FreeSpace=%FreeSpace:~-15%"
+
+Echo FreeSpace="%FreeSpace%"
+Echo    100 GB="%GB100%"
+
+If "%FreeSpace%" gtr "%GB100%" (
+  Echo yes enough free space
+) else (
+  Echo not enough free space - 100GB 
+  exit 5
+)
+
+REM Install missing dependencies
+powershell -command "& { iwr https://raw.githubusercontent.com/electron/electron/%CHECK_VERSION%/script/setup-win-for-dev.bat -OutFile C:\install.bat }"
+cmd /c C:\install.bat
+
+
+set GN_CONFIG=release
+
+echo "Building $env:GN_CONFIG build"
+SET PATH=%PATH%;C:\depot_tools\
+SET PATH=%PATH%;C:\Python27\
+git config --global core.longpaths true
+if not exist %buildir% mkdir %buildir%
+cd %buildir%
+set CHROMIUM_BUILDTOOLS_PATH=%buildir%\src\buildtools
+set SCCACHE_PATH=%buildir%\src\electron\external_binaries\sccache.exe
+REM set GYP_MSVS_OVERRIDE_PATH=C:\BuildTools\
+set GYP_MSVS_VERSION=2019
+set DEPOT_TOOLS_WIN_TOOLCHAIN=0
+REM set GCLIENT_EXTRA_ARGS="%GCLIENT_EXTRA_ARGS% --custom-var=\"checkout_boto=True\" --custom-var=\"checkout_requests=True\""
+cmd /C gclient config --name "src\electron" --unmanaged %GCLIENT_EXTRA_ARGS% "https://github.com/electron/electron"
+cmd /C gclient sync -f --with_branch_heads --with_tags
+cd src
+cd electron
+git fetch
+git checkout %CHECK_VERSION%
+cmd /c gclient sync -f
+cd ..
+cd ..
+set BUILD_CONFIG_PATH="//electron/build/args/release.gn"
+REM build out/Default
+cd src
+REM XXX: leave extra space after GN_EXTRA_ARG
+set GN_EXTRA_ARG= 
+cmd /C gn gen out/Default --args="import(\"//electron/build/args/release.gn\") %GN_EXTRA_ARG% cc_wrapper=\"%SCCACHE_PATH%\""
+ninja -C out/Default electron:electron_app
+cmd /C gn gen out/ffmpeg "--args=import(\"//electron/build/args/ffmpeg.gn\") %GN_EXTRA_ARGS%"

```

Comment:
This script seems to be inconsistent w.r.t `GN_EXTRA_ARG` vs `GN_EXTRA_ARGS`. Are either of these variables needed?